### PR TITLE
[JENKINS-65593] Fix test error for Jenkins >= 2.281

### PR DESCRIPTION
**Pull request by MRamonLeon**
# Description

See [JENKINS-65593](https://issues.jenkins-ci.org/browse/JENKINS-65593).
It fixes a test failure for Jenkins over or equal to 2.281.

I can't request reviewers. Pinging people:
- @car-roll 
- @olamy 
- @dwnusbaum 
- @joseblas 
 
# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeTest.java [hunk 1]
```diff
@@ -2291,7 +2292,12 @@ public void parameterizedPipeline() throws Exception {
         Assert.assertEquals("param2", parameters.get(1).get("name"));
         Assert.assertEquals("StringParameterDefinition", parameters.get(1).get("type"));
         Assert.assertEquals("string param", parameters.get(1).get("description"));
-        Assert.assertNull(((Map) parameters.get(1).get("defaultParameterValue")).get("value"));
+
+        if (Jenkins.getVersion().isOlderThan(new VersionNumber("2.281"))) {
+            Assert.assertNull(((Map) parameters.get(1).get("defaultParameterValue")).get("value"));
+        } else {
+            Assert.assertEquals("", ((Map) parameters.get(1).get("defaultParameterValue")).get("value"));
+        }
```
Comment (jglick) [inline blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeTest.java:2300]: or

```suggestion
        Assert.assertNull(Util.fixEmpty(((Map) parameters.get(1).get("defaultParameterValue")).get("value")));
```

**Comment by jtnord** [comment]
or should this actually be fixed in core?
jenkinsci/jenkins#5481

File: blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeTest.java (hunk 1 above)
Comment (MRamonLeon) [reply blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeTest.java:2300]: I wanted to leave the difference clear. To get rid of what's going to happen with the core PR, this approach is better.

**Comment by MRamonLeon** [comment]
I think we can merge it independently of what will happen with the [Core PR](https://github.com/jenkinsci/jenkins/pull/5481)

@car-roll , @bitwiseman , @olamy could you merge if you agree please?

**Comment by MRamonLeon** [comment]
+ release if possible. Thank you.

**Pull request merged**